### PR TITLE
Update oomkill.py

### PR DESCRIPTION
**Pull request by smita-koralahalli**
The bcc tool available till date supports from kernel version 4.4 to 4.8 and gives anamolous output as given below for OOM kill of PID and total_pages consumed

17:39:52 Triggered by PID 10505 ("a.out"), OOM kill of PID 1835561760 ("lukB high:%lukB "), 2361183241434822607 pages, loadavg: 1.88 1.70 1.24 3/479 10508.

As mentioned iovisor#1911 function oom_kill_process has been changed after kernel version 4.8 and these few changes in the bcc tool works fine for the kernel till date.

Output after modification has been given below!
23:38:03 Triggered by PID 22426 ("a.out"), OOM kill of PID 22426 ("a.out"), 4085393 pages, loadavg: 6.01 5.09 4.22 1/715 23199

### CODE REVIEW

**Comment by yonghong-song** [comment]
[buildbot, test this please]

**Comment by yonghong-song** [comment]
could you move old oomkill.py to tools/old directory since it may still be used for 4.4 kernel?

File: tools/oomkill.py [hunk 1]
```diff
@@ -36,14 +36,16 @@
 
 BPF_PERF_OUTPUT(events);
 
-void kprobe__oom_kill_process(struct pt_regs *ctx, struct oom_control *oc,
-    struct task_struct *p, unsigned int points, unsigned long totalpages)
+void kprobe__oom_kill_process(struct pt_regs *ctx, struct oom_control *oc, const char *message)
 {
+    struct task_struct *chosen;
```
Comment (palmtenor) [inline tools/oomkill.py]: This line is not used?

**Comment by smita-koralahalli** [comment]
Yes, I tested it works without that line and I have removed it

**Comment by yonghong-song** [comment]
@smita-koralahalli  could you move the old oomkill.py to tools/old directory as it may still be used for 4.4? Other than that, the patch looks good. Thanks!

**Comment by yonghong-song** [comment]
[buildbot, test this please]

**Comment by yonghong-song** [comment]
@smita-koralahalli i checked out the change. it looks like you just moved the oomkill.py to tools/old directory and made the change there. The following is the correct way to do it:
  - copy the oomkill.py to tools/old directory
  - modify the tools/oomkill.py with new function prototypes
So after your change, tools/old/oomkill.py is used for linux 4.4 and tools/oomkill.py is used for 4.9 and later kernels. hope this is clear.

**Comment by yonghong-song** [comment]
BTW, the py_test_tools_smoke failed on all platforms since it tests tools/oomkill.py and it is gone.

**Comment by smita-koralahalli** [comment]
Sorry for the confusion . I got that, I will be closing this pull request and raise a new one with the steps you mentioned.

**Pull request closed without merging**